packages/nimble-components/.babelrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100
+        }
+      }
+    ],
+    "@babel/preset-typescript"
```

Comment:
based on the discussion below that ts-loader is handling all the typescript work, I'm curious if this is needed.

edit: removed this preset locally and it still worked, think we can remove this and the npm dep 👍